Library/ViewModels/CommentRepliesViewModelTests.swift

```diff
@@ -34,4 +45,139 @@ internal final class CommentRepliesViewModelTests: TestCase {
       self.loadCommentIntoDataSourceComment.assertValue(rootComment)
     }
   }
+
+  func testOutput_ConfigureCommentComposerViewWithData_IsLoggedOut() {
```

Comment:
Try to specify an outcome in this test name and the ones below ie `_HasCommentComposer` or `_HasBlockedCommentComposer`, `_HasNoCommentComposer`